cmd/gvproxy/main.go

```diff
@@ -124,6 +140,9 @@ func main() {
 	if bessSocket != "" {
 		protocol = types.BessProtocol
 	}
+	if vfkitSocket != "" {
```

Comment:
this seems it can be combined with previous check

Reply:
I took a very basic approach, and added an additional block where there were pre-existing references to `qemuSocket` and `bessSocket`. All of this could probably be refactored/simplified, but I haven't thought about it.